features/solr/pom.xml

```diff
@@ -93,6 +93,36 @@
             <artifactId>admin-core-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.cxf</groupId>
+            <artifactId>cxf-rt-transports-http</artifactId>
+            <version>3.4.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>5.4.24.Final</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-hdfs-client</artifactId>
```

Comment:
❗ I don't think we use this with our Solr client. Can we suppress these other transitive dependencies instead?

Here are a couple of other examples of us doing that.
https://github.com/codice/ddf/blob/master/platform/solr/solr-dependencies/pom.xml#L62-L72
https://github.com/codice/ddf/blob/master/platform/solr/solr-dependencies/pom.xml#L93-L122